### ISSUE TITLE
Add to Julia Registry and update docs

### ISSUE THREAD

**Issue by phaverty** [GenieFramework/Genie.jl#138]
**Describe the bug**
The README instructs users to install Genie using the GitHub URL, rather than the julia registry. I see that there is a Genie release for the julia registry. When I first read the README, I assumed Genie was still a prototype and that it wasn't registered. Adding instructions for 

```
Pkg.add("Genie")
```
would give a bit of extra confidence in the solidity of the package.

**Comment by essenciary** [comment]
Hmmm - is Genie registered?! Can you provide more details? 
I submitted a PR to the registry a long time ago but it wasn't merged and I didn't follow up anymore. Maybe somebody merged it and I wasn't aware...

**Comment by phaverty** [comment]
Oh, looks like it isn't. I saw your GitHub release and thought it corresponded to a version in the Registry.
The mechanism for all of this changed recently. I stumbled my way through it to get my packages on the new system last week. Using these TagBot and Registrator GitHub bots made it fairly easy. 

https://discourse.julialang.org/t/switching-package-registration-systems-soon/22677/20
https://discourse.julialang.org/t/ann-tagbot-creates-tags-and-releases-for-your-julia-packages-when-theyre-registered/23084

There is reasonable documentation, but it is a bit scattered and hard to find. The above community discussions were very helpful.
